Building with -g on x86 can use more than 4G memory, so try to
lower the memory requirement by forcing ld(1) --no-keep-memory.

--- build/common.gypi.orig	2014-09-26 11:50:55.381056525 +0200
+++ build/common.gypi	2014-09-26 11:51:58.281560018 +0200
@@ -3287,6 +3287,7 @@
         'ldflags': [
           '-Wl,-z,now',
           '-Wl,-z,relro',
+	   '-Wl,--no-keep-memory',
         ],
       },
     }],
